src/dragonball/src/device_manager/balloon_dev_mgr.rs

```diff
@@ -275,6 +286,13 @@ impl Default for BalloonDeviceMgr {
     }
 }
 
+impl Drop for BalloonDeviceMgr {
+    fn drop(&mut self) {
+        // todo move METIRCS oprations to remove_device?
```

Comment:
finish your todo

Reply:
The balloon device metric should be removed in device_manager since it's added in device_manager. But the device_manager doesn't have the `remove_device` function. So we could remove the metrics here.

Reply:
Done.